tests/cbor_decode/test8_suit12/src/main.c

```diff
@@ -0,0 +1,827 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+#include <ztest.h>
+#include "manifest12_decode.h"
+#include "cbor_debug.h" // Enables use of print functions when debugging tests.
+
+uint8_t crash_vector0[] = {
```

Comment:
Maybe a comment on these to describe where they come from?

Reply:
Added

Reply:
Same, I'm not seeing the comment in the latest commit diff.